models/base/manifest/default/snowplow_media_player_base_sessions_lifecycle_manifest.sql

```diff
@@ -0,0 +1,114 @@
+{{
+  config(
+    materialized="incremental",
+    unique_key='session_id',
+    upsert_date_key='start_tstamp',
+    sql_header=snowplow_utils.set_query_tag(var('snowplow__query_tag', 'snowplow_dbt')),
+    partition_by = snowplow_utils.get_value_by_target_type(bigquery_val={
+      "field": "start_tstamp",
+      "data_type": "timestamp"
+    }, databricks_val='start_tstamp_date'),
+    full_refresh=snowplow_media_player.allow_refresh(),
```

Comment:
Double check some of these configs against the web package please, some of them are specific to a warehouse and I am not sure if you are maybe missing some (like sort and dist for redshift)

Reply:
I think that's my bad then there should have been more bespoke stuff per warehouse there... 
This is the generic one from web that covers all warehouse, obviously might need to adjust a few bits:
```
{{
  config(
    materialized='incremental',
    unique_key='session_id',
    upsert_date_key='start_tstamp',
    sort='start_tstamp',
    dist='session_id',
    partition_by = snowplow_utils.get_value_by_target_type(bigquery_val={
      "field": "start_tstamp",
      "data_type": "timestamp"
    }, databricks_val='start_tstamp_date'),
    cluster_by=snowplow_web.web_cluster_by_fields_sessions_lifecycle(),
    full_refresh=snowplow_web.allow_refresh(),
    tags=["manifest"],
    sql_header=snowplow_utils.set_query_tag(var('snowplow__query_tag', 'snowplow_dbt')),
    tblproperties={
      'delta.autoOptimize.optimizeWrite' : 'true',
      'delta.autoOptimize.autoCompact' : 'true'
    },
    snowplow_optimize = true
  )
}}
```

Can you also raise an issue in ecommerce to improve this there :D

Reply:
Have added the missing properties to the config and also raised a PR in [ecommerce here](https://github.com/snowplow/dbt-snowplow-ecommerce/pull/35).

Reply:
I just meant raise the issue not to also have you fix it, but thanks!